jenkins/model_output_processing_Jenkinsfile

```diff
@@ -27,7 +27,14 @@ pipeline {
         // Note that the Jenkins is on UTC!
         // May need to actually use yesterday's date depending on what time of day this runs
         //sh 'DATE=$(date -d "yesterday 13:00" "+%Y_%m_%d"); wget https://owi-common-resources.s3-us-west-2.amazonaws.com/resources/application/nhm/output/data/climate_${DATE}.nc'
+        sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/model_output/all_quantiles.rds'
```

Comment:
If you use three ticks, you can make a multi-line `sh` block, e.g. here: https://github.com/usgs-makerspace/wbeep-processing/blob/master/jenkins/tippecanoe_create_tiles_Jenkinsfile#L45

Reply:
Then you could reuse the date variable